frontend/src/components/Comment/PostComment.js

```diff
@@ -1,15 +1,23 @@
-import React from "react";
+import React, { useState }  from 'react'
 import "./CommentAPI";
 import { postComment } from "./CommentAPI";
-import "./PostComment.scss";
+import "./SCSS/PostComment.scss";
 
-class PostComment extends React.Component {
+export const PostComment = (props) => {
 
-    comment(e) {
+    const [textAreaText, settextAreaText] = useState("");
+
+    const handleTyping = (e) => {
+        settextAreaText(e.target.value);
+    }
+
+    //Post a comment
+    const comment = (e) => {
         e.preventDefault();
-        postComment(e.target.comment.value, this.props.postId).then(
+        postComment(e.target.comment.value, props.postId).then(
```

Comment:
since you are binding `textAreaText` to the value of the input, then you should probably do `postComment(textAreaText, ...)`. also, maybe it might be better to call the getter `gettextAreaText`?

Reply:
Sure, good suggestion, however I don't think there are getter for react `usestate`. So I'll leave it to `postComment(textAreaText, ...)`

Reply:
ah i see youre right